rust_icu_unum/src/lib.rs

```diff
@@ -200,6 +201,73 @@ impl UNumberFormat {
         )?;
         Ok((result, iterator))
     }
+
+    /// Implements `unum_formatDecimal`. Since 0.3.1.
```

Comment:
It's unfortunate that rustc doesn't allow `#[stable(since = "0.3.1")]` outside of the standard library.